analyzers/tests/SonarAnalyzer.UnitTest/TestCases/VariableShadowsField.cs

```diff
@@ -16,7 +16,7 @@ public class VariableShadowsField
         public void doSomething(object someParameter)
         {
             int myField = 0, // Noncompliant
-//              ^^^^^^^
+            //  ^^^^^^^
```

Comment:
This change was not needed.

Reply:
I prefer this style because it does not break if you do "Format Document".